hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -1115,15 +1115,20 @@ def test_basic_indices_replaces_whole_axis_slices_with_ellipsis(idx):
 @given(
     shape=nps.array_shapes(min_dims=0, max_side=4)
     | nps.array_shapes(min_dims=0, min_side=0, max_side=10),
-    min_dims=st.integers(0, 5),
-    allow_ellipsis=st.booleans(),
     allow_newaxis=st.booleans(),
+    allow_ellipsis=st.booleans(),
     data=st.data(),
 )
 def test_basic_indices_generate_valid_indexers(
-    shape, min_dims, allow_ellipsis, allow_newaxis, data
+    shape, allow_newaxis, allow_ellipsis, data
```

Comment:
I wouldn't bother changing this now, but usually better style to avoid this kind of diff-without-a-difference 😉